.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
The prettier options are already listed in the eslint file.

Reply:
eslint is not respected in vscodes prettier command,
![image](https://user-images.githubusercontent.com/17952364/134497464-09606dce-5b53-4dcf-9445-6cb0de1823cd.png)

if you format with vscode prettierrc is respected

Reply:
That format document misses all eslint rules. Therefore, it is still better to use eslint to fix all problems instead.

Reply:
okay, removed it